api/openapi-spec/openapi.yaml

```diff
@@ -347,12 +340,12 @@ components:
           type: string
         client_secret:
           type: string
-  securitySchemes:
-    SdkKeyAuth:
-      in: header
-      name: X-Optimizely-SDK-Key
-      type: apiKey
-    TokenAuth:
-      type: http
-      scheme: bearer
-      bearerFormat: JWT
+# securitySchemes:
```

Comment:
Python lib not supporting securitySchemes, that's why commented out.

Reply:
A bit unfortunate to remove this.

Reply:
Can you add a comment above this about why it is still around?

Reply:
It's was mistake in our code, didn't realize. fixed the issue.